src/base-command.ts

```diff
@@ -78,9 +78,12 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
 
     this.validateRegionAndAuth();
 
-    this.developerHubBaseUrl =
-      this.sharedConfig.developerHubBaseUrl || this.developerHubUrl;
-    if (!this.developerHubBaseUrl) this.developerHubBaseUrl = getDeveloperHubUrl();
+    this.developerHubBaseUrl = this.sharedConfig.developerHubBaseUrl;
+    if (this.developerHubUrl && this.developerHubUrl.startsWith("https://")) {
+      this.developerHubBaseUrl = this.developerHubUrl.replace("https://", "");
+    }
```

Comment:
@harshithad0703 , why have you added this condition?

Reply:
this.developerHubBaseUrl expects the host name only, this.developerHubUrl provides along with protocol (https)

Reply:
@harshithad0703, Instead of handling like this, try to handle this way
    if (this.developerHubUrl?.startsWith("https")) {
      this.developerHubBaseUrl = this.developerHubUrl?.split("//")[1];
    }

Reply:
done